frontend/test/metabase/lib/expressions/resolver.unit.spec.js

```diff
@@ -170,6 +170,31 @@ describe("metabase-lib/v1/expressions/resolve", () => {
       it("should allow substring with index=1", () => {
         expect(() => expr(["substring", "foo", 1, 1])).not.toThrow();
       });
+
```

Comment:
Let's also add an explicit repro for the app crash?
BTW there's a GH issue for this bug: #53682

Reply:
Added a e2e repro